practice/bio/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+
+<html lang= "en">
+
+  <head>
+    <meta charset="utf-8">
+    <title>Bio</title>
+    <link rel="stylesheet" href="site.css" />
+  </head>
+
+  <body>
+    <header>
+      <h1>John Cena Bio</h1>
+    </header>
+    <nav>
+    </nav>
+    <section>
+      <p>
+        John Felix Anthony Cena Jr. (born April 23, 1977) is an American professional wrestler, rapper, actor, and reality television show host signed to WWE on the SmackDown brand.
+      </p>
+      <img src="johncena.jpg">
+    </section>
+    <section>
+      <h2>Places lived in</h2>
+      <p>
+        <ul>
+          <li>West Newburry, Massachusetts</li>
+          <li>Tampa, Florida</li>
+        </ul>
+      </p>
```

Comment:
Block elements are not allowed to be in paragraph `p` elements. This will actually be parsed as:

```html
<p></p>
<ul>
...
</ul>
<p></p>
```